DeepCat.js

```diff
@@ -8,7 +8,8 @@
 ( function( $, mw ) {
 	var DeepCat = {},
 		keyString = 'deepcat:',
-		maxDepth = 15,
+		defaultDepth = 15,
+		maxDepth = 20,
```

Comment:
WTF? I'm really pissed now. Do you read your own comments? You demanded, that I build back in the maximal limit. I explained several times (online & offline) that I don't think a maximal limit is needed in the front-end because there is one in the backed and there is an additional limit on the total number of categories. You repeatedly said you want still an maximal number in the deep-cat gadget, so I build it in for you! Now you criticize my new patch for doing exactly what you "forced" me to implement and you propose exactly my original solution?
Forget about me as an working-student for a moment (at least the original patch was done in my free time) – as an volunteer I would probably not contribute ever again to a WMDE project. Volunteer support done right. Not.